Automattic/jetpack
# Contact Form: center submit button content horizontally

Fixes https://github.com/Automattic/jetpack/issues/35268

## Proposed changes:

Aligns the content of the _Contact Form_ submit button horizontally. The alignment issue was visible when the width of the button was set to 100%.

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Spin up a test site with this branch
- Build the package: `cd projects/packages/forms && npm run build`
- Create a new post
- Add a _Contact Form_ block
- Set the width of the submit button to 100%
- Notice the content is centered
- Open the preview
- Check the content is centered as well

<img width="400" alt="Screenshot 2024-01-29 at 2 37 44 PM" src="https://github.com/Automattic/jetpack/assets/1620183/a27f9af5-8917-4ced-ba6c-37d91db1f3fb">
